### ISSUE TITLE
Some Users Block From Loading 0c222aae66.css

### ISSUE THREAD

**Issue by trismart-smartboard** [FortAwesome/Font-Awesome#18519]
I have several users in Texas who are being blocked from loading https://use.fontawesome.com/0c222aae66.css, so their icons don't load. When I have them enter that URL directly into their browsers, it returns "This site can't be reached...The connection was reset".

Is there an issue with the way this file is being served? Do you have any idea why this is happening?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Let's ask @robmadole

**Comment by robmadole** [comment]
Hi @trismart-smartboard that sounds like it may be firewall related. Are these folks in corporate networks? Can you figure out any commonality between them?

**Comment by trismart-smartboard** [comment]
Our IT lead says that Meraki is classifying Fontawesome as a malware site.

Thanks!

[image: ] *How to Report a Problem*
<https://mailchi.mp/35f630f853b4/reporting-errors-in-smartboard-4880580?e=746c9fc56d>
 [image: ]



*Jay, SMARTBoard Project Manager*
phone: 801-668-4501 <+18016684501>
email: ***@***.***
web: SMARTBoardCRM.com <https://www.smartboardcrm.com/>

**Comment by trismart-smartboard** [comment]
Hi, Rob,

See my previous email.

Thanks!

[image: ] *How to Report a Problem*
<https://mailchi.mp/35f630f853b4/reporting-errors-in-smartboard-4880580?e=746c9fc56d>
 [image: ]



*Jay, SMARTBoard Project Manager*
phone: 801-668-4501 <+18016684501>
email: ***@***.***
web: SMARTBoardCRM.com <https://www.smartboardcrm.com/>

**Comment by robmadole** [comment]
@trismart-smartboard yep. this happens every once in a while. we show up as a false positive and get blocked. There isn't a whole lot we can do here. (It's not us that have made an error.) If you can provide a link to removing false positives I'm happy to fill out a form.

**Comment by trismart-smartboard** [comment]
Adding Elisa Cisneros. See Fontawesome's email below.

Thanks!

[image: ] *How to Report a Problem*
<https://mailchi.mp/35f630f853b4/reporting-errors-in-smartboard-4880580?e=746c9fc56d>
 [image: ]



*Jay, SMARTBoard Project Manager*
phone: 801-668-4501 <+18016684501>
email: ***@***.***
web: SMARTBoardCRM.com <https://www.smartboardcrm.com/>
